tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,145 @@
+package e2e
+
+import (
+	"context"
+	"crypto"
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/drbg"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/mathrand"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
+	memorySigner "github.com/oasisprotocol/oasis-core/go/common/crypto/signature/signers/memory"
+	oasisGrpc "github.com/oasisprotocol/oasis-core/go/common/grpc"
+	"github.com/oasisprotocol/oasis-core/go/common/quantity"
+	"github.com/oasisprotocol/oasis-core/go/oasis-node/cmd/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/oasisprotocol/oasis-core/go/common/entity"
+	consensus "github.com/oasisprotocol/oasis-core/go/consensus/api"
+	"github.com/oasisprotocol/oasis-core/go/consensus/api/transaction"
+
+	"github.com/oasisprotocol/oasis-core/go/oasis-node/cmd/debug/txsource/workload"
+	staking "github.com/oasisprotocol/oasis-core/go/staking/api"
+	v1 "github.com/oasisprotocol/oasis-indexer/api/v1"
+	"github.com/oasisprotocol/oasis-indexer/tests"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const (
+	fundAccountAmount = 10000000000
+	timeout           = 1 * time.Second
+)
+
+func TestIndexer(t *testing.T) {
+	if _, ok := os.LookupEnv("OASIS_INDEXER_E2E"); !ok {
+		t.Skip("skipping test since e2e tests are not enabled")
+	}
+
+	if err := common.Init(); err != nil {
+		common.EarlyLogAndExit(err)
+	}
+	tests.Init()
+
+	// Indexer should be up
+	var status v1.Status
+	err := tests.GetFrom("/", &status)
+	require.Nil(t, err)
+
+	connString := "unix:/testnet/net-runner/network/validator-0/internal.sock"
+	conn, _ := oasisGrpc.Dial(connString, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
```suggestion
	conn, err := oasisGrpc.Dial(connString, grpc.WithTransportCredentials(insecure.NewCredentials()))
	require.Nil(t, err)
```
any reason not to do this (and same for all other variables for which blank identifier is used below)?

Reply:
👌